Formula/fq.rb

```diff
@@ -17,11 +17,11 @@ class Fq < Formula
 
   depends_on "concurrencykit"
   depends_on "jlog"
-  depends_on "openssl@1.1"
 
   uses_from_macos "sqlite"
 
   on_linux do
+    depends_on "openssl@3"
```

Comment:
Doesn't seem to be used on macOS. 
- [Makefile](https://github.com/circonus-labs/fq/blob/master/Makefile) adds `-lcrypto` to SunOS, Linux, and FreeBSD.
- [fqd_ccs.c](https://github.com/circonus-labs/fq/blob/master/fqd_ccs.c#L38-L55) has `__MACH__` specific code while else section is for OpenSSL.